lib/PuppeteerSharp/ViewPortOptions.cs

```diff
@@ -45,5 +45,14 @@ public ViewPortOptions()
         /// </summary>
         /// <value>Specifies if viewport supports touch events. Defaults to <c>false</c>.</value>
         public bool HasTouch { get; set; }
+
+        /// <summary>
+        /// Default Viewport
+        /// </summary>
+        public static ViewPortOptions Default => new ViewPortOptions
```

Comment:
make readonly

Reply:
But it's a getter. It is read-only. Am I missing something?

Reply:
Oops